packages/react-charting/src/components/SankeyChart/SankeyChart.base.tsx

```diff
@@ -68,34 +131,197 @@ export class SankeyChartBase extends React.Component<
         role={'presentation'}
         ref={(rootElem: HTMLDivElement) => (this.chartContainer = rootElem)}
       >
-        <svg width={width} height={height} id={getId('sankeyChart')}>
-          <g className={this._classNames.nodes}>{nodeData}</g>
-          <g className={this._classNames.links} strokeOpacity={0.2}>
-            {linkData}
-          </g>
-        </svg>
+        <FocusZone direction={FocusZoneDirection.bidirectional} isCircularNavigation={true} allowTabKey={true}>
+          <svg width={width} height={height} id={getId('sankeyChart')}>
+            <g className={this._classNames.links} strokeOpacity={1}>
+              {linkData}
+            </g>
+            <g className={this._classNames.nodes}>{nodeData}</g>
+          </svg>
+        </FocusZone>
       </div>
     );
   }
 
+  /**
+   * This is used for calculating the node non normalized value based on link non normalized value.
+   *
+   */
+  private _populateNodeActualValue(data: ISankeyChartData) {
+    data.links.forEach((link: SLink) => {
+      if (!link.originalValue) {
+        link.originalValue = link.value;
+      }
+    });
+    data.nodes.forEach((node: SNode) => {
+      node.actualValue = Math.max(
```

Comment:
node's value is determined by the max of the inflow and outflow as per d3 sankey(https://github.com/d3/d3-sankey/blob/bb7233ea0004669878d922e55fe83084ef3cb555/src/sankey.js#L152).
Since we have to normalize the values to accommodate 1% of the total height changes and node.value is calculated by d3 sankey based on normalized values of links. In order to show user the unnormalized value we need to calculate node's unnormalized value using links unnormalized value , technically the max of (sum of incoming links normalized value, sum of outgoing link normalized value).